src/app/app.component.html

```diff
@@ -57,5 +57,18 @@
   >Pandu</a>
   (<a href="https://github.com/prendradjaja/covid-dashboard" target="_blank" rel="noopener noreferrer">repo</a>)
 
-  <span class="git-version">&mdash; {{version.commit}}{{version.untrackedMark}} ({{version.branch}})</span>
+  <span class="developer-info">
+    &mdash;
+    {{version.commit}}{{version.untrackedMark}} ({{version.branch}})
+    &mdash;
+    <a href="http://localhost:4200/#{{ route.fragment | async }}"
+      target="_blank"
+      rel="noopener noreferrer"
+    >Local</a>
+    /
+    <a href="https://personal-covid-dashboard.web.app/#{{ route.fragment | async }}"
+      target="_blank"
+      rel="noopener noreferrer"
+    >Hosted</a>
```

Comment:
'Local' and 'Hosted' links. Also changed link blue slightly from previous commit :P

<img width="719" alt="Screen Shot 2020-04-16 at 8 33 45 PM" src="https://user-images.githubusercontent.com/3111845/79529489-a3e21400-8021-11ea-97b1-1f1ea9deac90.png">